app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -143,12 +143,10 @@ public View getView(int pos, @Nullable View convertView, @NonNull ViewGroup pare
             final TextView textView;
             if (convertView == null) {
                 textView = (TextView) _inflater.inflate(_layout, parent, false);
-                textView.setBackgroundResource(R.drawable.search_dialog_selection);
```

Comment:
@gsantner This needs to be reverted. Without the border-only highlight, the visibility of highlighted text against the selected background is very poor.

Reply:
default android behaviour is way better to understand what is highlighted and what not to me:

![screenshot-2021-07-14__18-04-55](https://user-images.githubusercontent.com/6735650/125654847-3707be16-1576-4c90-9c8d-29b915f5e92b.png)


![screenshot-2021-07-14__18-04-41](https://user-images.githubusercontent.com/6735650/125654839-9c1b894e-9c1b-4a33-91be-0199aeb31e20.png)

Reply:
![image](https://user-images.githubusercontent.com/1940940/125671610-756eb27a-0088-4aff-a3a9-c04d23a50da0.png)

Issue is when highlighting is present

Reply:
I'd rather pefer to make the text bold, or i.e. accent colored

Or/And disable syntax highlighting for selected elements

Reply:
Of these, the checkmark icon is the cleanest solution, IMO. I will work on a PR for it.